instance-conf/httpd-2.4/bin/httpdenv.sh

```diff
@@ -14,7 +14,7 @@ export LD_LIBRARY_PATH
 OPENSSL_CONF=@@PRODUCT_ROOT@@/httpd-2.4/ssl/openssl.cnf
 export OPENSSL_CONF
 
-MANPATH=@@PRODUCT_ROOT@@/httpd-2.4/man:$MANPATH
+MANPATH=@@PRODUCT_ROOT@@/httpd-2.4/man${MANPATH:+:}$MANPATH
```

Comment:
perhaps functionally accurate, but looks odd.  Maybe move the latter MANPATH into the curlies?

Reply:
Actually, identical to existing line 11 above. I'd keep it to be as simplistic as possible there, and leave all the complexity of choosing a ':' in the complex expansion preceding it.